packages/veritone-react-common/src/components/FormBuilder/FormItems/CheckBoxes.js

```diff
@@ -32,14 +32,20 @@ export default function Checkboxes({
   }, [name, value, onChange])
 
   return (
-    <FormControl error={error.length > 0} fullWidth>
+    <FormControl
+      error={error.length > 0}
```

Comment:
Could error be undefined here?

Reply:
all error is string and have default value to ''